src/Microsoft.Health.Fhir.Core/Features/Operations/Import/ImportResourceLoader.cs

```diff
@@ -151,6 +151,11 @@ private ImportResource ParseImportRawContent(string resourceType, (string Conten
                 {
                     throw new FormatException("Resource type not match.");
                 }
+
+                if (!string.IsNullOrEmpty(resourceType) && resourceType.Equals("SearchParameter", StringComparison.Ordinal))
+                {
+                    throw new ArgumentException("SearchParameter is not a supported resource type.");
```

Comment:
NIT - maybe something like `SearchParameter resources cannot be processed by import.`